.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+ignore = E203 E501 W503 W504
```

Comment:
Great question,. Black recommends ignoring certain flake8 warnings as they aren't actually pep8 compliant. Since black automatically wraps lines (with some flexibility) there's no need to actually check for long lines hence why I've also added E501.

I just learned about B950 via flake8-bugbear, which might be interesting to add. We can save that for a different PR: https://github.com/PyCQA/flake8-bugbear#opinionated-warnings

Reply:
Bump this to a different PR.